x-pack/plugins/data_visualizer/public/application/common/components/stats_table/components/field_data_expanded_row/keyword_content.tsx

```diff
@@ -55,6 +55,16 @@ export const KeywordContent: FC<FieldDataRowProps> = ({ config, onAddFilter }) =
         barColor="success"
         onAddFilter={onAddFilter}
       />
+      {config.stats?.sampledValues && fieldName !== undefined ? (
+        <TopValues
+          stats={stats}
+          fieldFormat={fieldFormat}
+          barColor="success"
+          onAddFilter={onAddFilter}
+          showSampledValues={true}
```

Comment:
I wonder if it's worth hiding this when we're in the ML view?

Reply:
As discussed in Slack, we can keep this table here and tweak the labels for more clarity.